frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -310,50 +310,63 @@ export const FieldRowContainer = ({
 
 type MemoFieldRowProps = {
   field: FormFieldDto
-  questionNumber?: string
+  questionNumber?: number
 }
 
-const MemoFieldRow = memo(({ field, ...rest }: MemoFieldRowProps) => {
-  switch (field.fieldType) {
-    case BasicField.Attachment:
-      return <AttachmentField schema={field} {...rest} />
-    case BasicField.Checkbox:
-      return <CheckboxField schema={field} {...rest} />
-    case BasicField.Mobile:
-      return <MobileField schema={field} {...rest} />
-    case BasicField.HomeNo:
-      return <HomeNoField schema={field} {...rest} />
-    case BasicField.Email:
-      return <EmailField schema={field} {...rest} />
-    case BasicField.Nric:
-      return <NricField schema={field} {...rest} />
-    case BasicField.Number:
-      return <NumberField schema={field} {...rest} />
-    case BasicField.Decimal:
-      return <DecimalField schema={field} {...rest} />
-    case BasicField.Date:
-      return <DateField schema={field} {...rest} />
-    case BasicField.Dropdown:
-      return <DropdownField schema={field} {...rest} />
-    case BasicField.Statement:
-      return <ParagraphField schema={field} {...rest} />
-    case BasicField.ShortText:
-      return <ShortTextField schema={field} {...rest} />
-    case BasicField.LongText:
-      return <LongTextField schema={field} {...rest} />
-    case BasicField.Radio:
-      return <RadioField schema={field} {...rest} />
-    case BasicField.Rating:
-      return <RatingField schema={field} {...rest} />
-    case BasicField.Section:
-      return <SectionFieldRow field={field} {...rest} />
-    case BasicField.Uen:
-      return <UenField schema={field} {...rest} />
-    case BasicField.YesNo:
-      return <YesNoField schema={field} {...rest} />
-    case BasicField.Image:
-      return <ImageField schema={field} {...rest} />
-    case BasicField.Table:
-      return <TableField schema={field} {...rest} />
-  }
-})
+const MemoFieldRow = memo(
+  ({ field, questionNumber, ...rest }: MemoFieldRowProps) => {
+    // Q: The generation of question numbers is done way upstream and produces a
```

Comment:
nit: should we remove this comment, since it may add confusion in the future?

Reply:
I'll rephrase the comment to be a note for possible improvement rather than a question!